tools/jsrun.py

```diff
@@ -30,7 +30,7 @@ def make_command(filename, engine=None, args=[]):
   # label a path to nodejs containing a 'd8' as spidermonkey instead.
   jsengine = os.path.split(engine[0])[-1]
   # Use "'d8' in" because the name can vary, e.g. d8_g, d8, etc.
-  return engine + [filename] + (['--'] if 'd8' in jsengine or 'jsc' in jsengine else []) + args
+  return engine + [filename] + (['--expose-wasm', '--'] if 'd8' in jsengine or 'jsc' in jsengine else []) + args
```

Comment:
Yeah, and the flag has been in d8 for several months, so I don't think it should be a big issue